src/VisualStudio/Core/Def/InheritanceMargin/InheritanceMarginTag.cs

```diff
@@ -41,27 +45,17 @@ public InheritanceMarginTag(Workspace workspace, int lineNumber, ImmutableArray<
             Workspace = workspace;
             LineNumber = lineNumber;
             MembersOnLine = membersOnLine;
+
             // The common case, one line has one member, avoid to use select & aggregate
```

Comment:
I guess technically you are still avoiding the use of select & aggregate, but I'm not sure how useful of a comment it is.

```suggestion
            // The common case is that one line has one member.
```